sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -18,8 +20,44 @@ export interface AccessPolicy {
     start: Date;
 }
 
+// @public
+export interface AccountSASOptions {
+    expiresOn?: Date;
+    ipRange?: SasIPRange;
```

Comment:
It feels strange that `SAS` isn't all capitalized here like it is elsewhere, especially when it's right next to `IP` which is.

I have a small preference for not capitalizing acronyms in type names, e.g. `SasProtocol`, `AccountSasPermissions`, but I understand we might want to align with the other storage packages